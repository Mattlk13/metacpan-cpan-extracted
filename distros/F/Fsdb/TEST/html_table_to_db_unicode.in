<head>
<meta charset="UTF-8">
</head>
<body>
<table>
<tr><th>account</th> <th>passwd</th> <th>uid</th> <th>gid</th> <th>fullname</th> <th>homedir</th> <th>shell</th> </tr>
<tr bgcolor="#f0f0f0"><td>root</td> <td>*</td> <td>0</td> <td>0</td> <td>Root</td> <td>/root</td> <td>/bin/bash</td> </tr>
<tr bgcolor="#f0f0f0"><td>jose</td> <td>*</td> <td>2274</td> <td>134</td> <td>José</td> <td>/home/johnh</td> <td>/bin/bash</td> </tr>
<tr bgcolor="#f0f0f0"><td>lin</td> <td>*</td> <td>2275</td> <td>134</td> <td>林</td> <td>/home/greg</td> <td>/bin/bash</td> </tr>
</table>
</body>
