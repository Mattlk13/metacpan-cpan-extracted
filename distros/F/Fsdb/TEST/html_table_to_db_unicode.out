#fsdb -F S account passwd uid gid fullname homedir shell
root  *  0  0  Root  /root  /bin/bash
jose  *  2274  134  José  /home/johnh  /bin/bash
lin  *  2275  134  林  /home/greg  /bin/bash
#   | html_table_to_db
